demo/src/features/skybox/jobs/extract.rs

```diff
@@ -1,15 +1,16 @@
 use rafx::render_feature_extract_job_predule::*;
 
 use super::*;
-use rafx::assets::{AssetManagerRenderResource, ImageAsset, MaterialAsset};
-use rafx::base::resource_map::ReadBorrow;
+use rafx::assets::{AssetManagerExtractRef, AssetManagerRenderResource, ImageAsset, MaterialAsset};
 use rafx::distill::loader::handle::Handle;
+use std::marker::PhantomData;
 use std::sync::Arc;
 
 pub struct SkyboxExtractJob<'extract> {
-    asset_manager: ReadBorrow<'extract, AssetManagerRenderResource>,
+    asset_manager: AssetManagerExtractRef,
     skybox_material: Handle<MaterialAsset>,
     skybox_texture: Handle<ImageAsset>,
+    phantom_data: PhantomData<&'extract ()>,
```

Comment:
Is this `phantom_data` needed? Other jobs don't have it.

Reply:
it's needed because the 'extract lifetime was unused. There may be a way to remove the lifetime, but I felt that consistency/ease of adding it back was better.